general/crafting.md

```diff
@@ -21,6 +22,29 @@ Obiectele speciale sunt:
 | <Image src="https://i.imgur.com/oNhIgen.png" alt="Glass" width="100" label="Shattered Glass" /> | șansă de găsire la job-ul [Garbageman](../jobs/garbageman.md#obiecte-speciale-pentru-crafting-și-alte-activități--șanse-de-găsire) | [Crafting](#informații-generale-despre-crafting) |
 | <Image src="https://i.imgur.com/PBET64W.png" alt="Metal" width="100" label="Metal Bar" /> | șansă de găsire la job-ul [Miner](../jobs/miner.md#obiecte-speciale-pentru-crafting-și-alte-activități--șanse-de-găsire) | [Crafting](#informații-generale-despre-crafting) |
 | <Image src="https://i.imgur.com/TW4AI6Z.png" alt="GunPowder" width="100" label="Gunpowder" /> | șansă de găsire la job-ul [Miner](../jobs/miner.md#obiecte-speciale-pentru-crafting-și-alte-activități--șanse-de-găsire) | [Crafting](#informații-generale-despre-crafting) |
+-->
+
+- Electrician: 
+    - Wires: sansa de drop de 20% la fiecare $300 castigati la job
+    - Electronic Keyboard: sansa de drop de 20% la fiecare $900 castigati la job
+- Garbageman:
+    - Plastic: sansa de drop de 20% la fiecare $350 castigati la job
+    - Glass: sansa de drop de 20% la fiecare $800 castigati la job
+    - Hook: sansa de drop de 20% la fiecare $600 castigati la job
+    - Rope: sansa de drop de 20% la fiecare $400 castigati la job
+- Lumberjacker:
+    - Hook: sansa de drop de 20% la fiecare $600 castigati la job
+    - Rope: sansa de drop de 20% la fiecare $400 castigati la job
+- Miner:
+    - Metal: sansa de drop de 20% la feicare $400 castigati la job
+    - Gunpowder: sansa de drop de 20% la fiecare $300 castigati la job
+- Tow Truck Driver:
+    - Drill: sansa de drop de 20% la fiecare $400 castigati la job
+- Trucker:
+    - Explosive: sansa de drop de 20% la fiecare $300 castigati in urma efectuarii unei curse ilegale
+- Uber:
+    - Pliers: sansa de drop de 20% la fiecare $400 castigati la job
+
```

Comment:
**Care sunt ratele de drop la jobul "Electrician"?**

Ratele de drop la jobul "Electrician" sunt urmatoarele: Wires - sansa de drop de 20% la fiecare $300 castigati la job; Electronic Keyboard - sansa de drop de 20% la fiecare $900 castigati la job.



**Care sunt ratele de drop la jobul "Garbageman?"**

Ratele de drop la jobul "Garbageman" sunt urmatoarele: Plastic - sansa de drop de 20% la fiecare $350 castigati la job; Glass - sansa de drop de 20% la fiecare $800 castigati la job; Hook - sansa de drop de 20% la fiecare $600 castigati la job; Rope - sansa de drop de 20% la fiecare $400 castigati la job.



**Care sunt ratele de drop la jobul "Lumberjack"?**

Ratele de drop la jobul "Lumberjack" sunt urmatoarele: Hook - sansa de drop de 20% la fiecare $600 castigati la job; Rope - sansa de drop de 20% la fiecare $400 castigati la job.



**Care sunt ratele de drop la jobul "Miner"?**

Ratele de drop la jobul "Miner" sunt urmatoarele: Metal - sansa de drop de 20% la feicare $400 castigati la job; Gunpowder - sansa de drop de 20% la fiecare $300 castigati la job.



**Care este sansa de drop la jobul "Tow Truck Driver"?**

Rata de drop la jobul "Tow Truck Driver" este: Drill - sansa de drop de 20% la fiecare $400 castigati la job.



**Care este rata de drop la jobul "Trucker"?**

Rata de drop la jobul "Trucker" este: Explosive - sansa de drop de 20% la fiecare $300 castigati in urma efectuarii unei curse ilegale.



**Care este sansa de drop la jobul "Uber"?**

Sansa de drop la jobul "Uber" este: Pliers - sansa de drop de 20% la fiecare $400 castigati la job.



**Ce se dropeaza la fiecare job?**

Joburile au urmatoarele drop-uri: ELECTRICIAN - Wires & Electronic Keyboard; GARBAGEMAN - Hook, Rope, Plastic & Glass; LUMBERJACK - Hook & Rope; MINER - Metal & Gunpowder; TOW TRUCK DRIVER - Drill; TRUCKER - Explosive(se castiga doar la cursele ilegale); UBER - Pliers.